assets/src/css/frontend/give-frontend.scss

```diff
@@ -36,3 +36,13 @@
 #give-receipt{
 	@import '../../../../blocks/components/container-placeholder-animation/style';
 }
+
+.give-grid-ie-utility {
```

Comment:
This part of CSS has been purposely added to this file because for some reason adding this in `_grid.scss` seems to have no effect. Debugging costed too much time therefore for now I'm switching to another task, this can be debugged later.